### DISCUSSION TITLE
colour wheel

### DISCUSSION THREAD

**Discussion by tsukinousagis** [AprilSylph/XKit-Rewritten#1565] [Suggestion Box]
idk if it's possible but a colour wheel for the text? so it's more than just those 7 colours tumblr gives us

**Comment by marcustyphoon** [comment]
I would love to do this one, but off the top of my head I think it's basically impossible for technical reasons (the post editor is really complicated; the fact that Quick Tags can work in it is sort of a stroke of luck). Might look into it at some point to see if I'm wrong, though.
